docs/spec_draft.md

```diff
@@ -666,6 +666,8 @@ lexicographic comparison on the (real, imaginary) pairs.
 // %result: [[5, 6], [7, 8]]
 ```
 
+&nbsp;[More Examples](../stablehlo/tests/interpret_max.mlir)
```

Comment:
One last change, here and below:
`../stablehlo/tests/interpret_maximum.mlir`

Reply:
my bad! Thanks!